src/pages/docs/[[...filePath]].tsx

```diff
@@ -0,0 +1,198 @@
+import renderToString from "next-mdx-remote/render-to-string";
+import hydrate from "next-mdx-remote/hydrate";
+import path from "path";
+import { getMdxPaths, MdxPath } from "@vulcanjs/mdx";
+import { Link as NextLink } from "@vulcanjs/next-material-ui";
+import { promises as fsPromises, lstatSync, existsSync } from "fs";
+import { List, ListItem, Link, Typography } from "@material-ui/core";
+import matter from "gray-matter";
+import { muiMdComponents } from "~/components/layout/muiMdComponents";
+import { MdxRemote } from "next-mdx-remote/types";
+
+// Define components to allow them in your mdx files
+// You can also replace HTML tags (components is passed to MDXProvider )
+// @see https://mdxjs.com/table-of-components
+const components = {
+  ...muiMdComponents,
+};
+
+const indexLink = (
+  <div style={{ maxWidth: "1000px" }}>
+    <Link href="/docs">
+      <Typography>Back to documentation index</Typography>
+    </Link>
+  </div>
+);
+
+const homeLink = (
+  <div style={{ margin: "32px auto", maxWidth: "1000px" }}>
+    <Link href="/">
+      <Typography>Back to home</Typography>
+    </Link>
+  </div>
+);
+
+const previousPageLink = (filePath: string) => {
+  let path = filePath;
+  // Delete the final '/' on the folder filePaths
+  if (path.slice(-1) === '/') {
+    path = path.slice(0, -1)
+  }
+  // If the previous page is /docs, doesn't return anything. See indexLink instead
+  if (path.split('/').length < 2) {
+    return (<>  </>);
```

Comment:
return null if you have nothing to return, here you return a React fragment which is not what you want